File: src/main/java/dominio/indicadores/ExpresionValor.java
package dominio.indicadores;

import dominio.empresas.Empresa;

import java.time.Year;

public class ExpresionValor implements Expresion{
	
	private int valor;
	
	public ExpresionValor(int valor) {
		this.valor = valor;
	}

	public int evaluarEn(Empresa empresa, Year anio) {
		return valor;
	}
	
}

File: src/main/java/dondeInvierto/dominio/indicadores/ExpresionValor.kt
package dondeInvierto.dominio.indicadores

import dondeInvierto.dominio.empresas.Empresa

import java.time.Year

class ExpresionValor(private val valor: Int) : Expresion {

    override fun evaluarEn(empresa: Empresa, anio: Year): Int = valor

}
